index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portifolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins:wght@200;300;400;500;700&family=Roboto:wght@100;300;500;700&display=swap">
+</head>
+<body>
+    <div class="head-section">
+        <header>
+         <nav>
+            <a class="logo" href="#">My logo</a>
+            <ul id="list-items">
+                <li><a href="#">Portifolio</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+          <img src="./images/union-img.png" alt="union-icon">
+         </nav>
+        </header>
+      <div class="bio">
+        <h1>I'm Magembe<br>        Glad to see you!
+        </h1>
+        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+        </p>
+        <h2>Let's connect</h2>
+        <ul class="icons">
+           <li>
+            <a href="#"><img class="social-icons" src="./images/twitter.png" alt="twitter"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/linkkedin.png" alt="linkedin"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/more.png" alt="more"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/git.png" alt="github"></a>
+           </li>
+           <li>
+            <a href="#"><img class="social-icons" src="./images/peace.png" alt="peace"></a>
+           </li>
+        </ul>
+      </div>  
+    </div>
```

Comment:
- [ ] Your website should look exactly like the design (e.g. font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Kindly fix the background color. :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![your](https://user-images.githubusercontent.com/70483835/220441125-b336cac5-0399-41b3-b7a5-7b1d4ed3f82a.png) | ![png](https://user-images.githubusercontent.com/70483835/220440763-f0f336b3-cba4-4d5b-9b5b-f389b1e112b5.png)